server/src/main/java/org/opensearch/common/io/stream/BufferedChecksumStreamInput.java

```diff
@@ -30,7 +30,7 @@
  * GitHub history for details.
  */
 
-package org.opensearch.index.translog;
```

Comment:
The split package issue (Java package `org.opensearch.comm` being used by both `:server` and `:libs/common`) is a problem and a blocker for adopting JPMS. I think we should avoid exacerbating that problem by adding more things in `org.opensearch.common` in the `:server` module. I suggest either not moving it, moving this elsewhere `:server`, or move it to `org.opensearch.core.common` in `:libs/core`.

Reply:
Shouldn't all commons go into libs/common?

Reply:
@Bukhtawar  Some of the base classes we use here are defined in `libs/core` which we cant access from `libs/common` without adding dependency.

Moved these classes to `:libs/core`